dali/image/tiff.cc

```diff
@@ -16,7 +16,7 @@
 
 namespace dali {
 
-namespace {
+namespace legacy_impl {
```

Comment:
```suggestion
namespace {
```
Since it's used only in this compilation unit

Reply:
not really. It is in the header `TiffBuffer` and used in tests

Reply:
done